cpu/sam0_common/periph/i2c.c

```diff
@@ -281,7 +283,7 @@ static int _start(SercomI2cm *dev, uint16_t addr)
 
     /* Wait for response on bus. */
     if (addr & I2C_READ) {
-        /* Some devices (e.g. SHT2x) can hold the bus while preparing the reply */
+        /* Some devices can hold the bus while preparing the reply */
```

Comment:
The example of SHT2x was added by me. If we leave it out there could be a moment in the future when somebody wants to know why.

Reply:
I'll change that.